.github/workflows/org-quality-gate.yml

```
@@ -0,0 +1,43 @@
+  actions: read
+
+jobs:
+  quality_and_security:
+    name: quality_and_security
+    if: ${{ hashFiles('.github/workflows/code-quality.yml') != '' }}
+    uses: ./.github/workflows/code-quality.yml
+    secrets: inherit
+
+  codeql:
+    name: codeql
+    if: ${{ hashFiles('.github/workflows/code-quality.yml') != '' }}
```

actionlint (1.7.11)

[error] calling function "hashFiles" is not allowed here. "hashFiles" is only available in "jobs.<job_id>.steps.continue-on-error", "jobs.<job_id>.steps.env", "jobs.<job_id>.steps.if", "jobs.<job_id>.steps.name", "jobs.<job_id>.steps.run", "jobs.<job_id>.steps.timeout-minutes", "jobs.<job_id>.steps.with", "jobs.<job_id>.steps.working-directory". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)